pkg/core/stateroot/module.go

```diff
@@ -181,6 +187,57 @@ func (s *Module) JumpToState(sr *state.MPTRoot) error {
 	return nil
 }
 
+const gcBatchSize = 100000
```

Comment:
`100_000`
Also if this is based on some calculations, worth a comment.

Reply:
After thinking for a while I've just removed it. 1M keys is ~33MB of memory, for 10K blocks interval we have 200-400K entries removed. Not something to worry about, especially since a single batch performs better (especially for Level).